src/locales/en.json

```diff
@@ -132,28 +111,29 @@
       "total-items": "How many new total items or new types of items can we bring to our users through this integration? Some sources are direct integrations, while others may be a source within a source."
     },
     "suggestions": "We appreciate suggestions for new sources from our community of users.",
-    "issue-button": "Suggest a new source"
+    "issue-button": "Suggest a new source",
+    "aria": { "table": "sources table" }
   },
   "meta-search-page": {
     "title": "Meta Search",
-    "intro": "Openverse is built on top of a catalog that indexes CC-licensed and public domain content from selected sources. Learn more about our {link}.",
+    "intro": "{openverse} is built on top of a catalog that indexes CC-licensed and public domain content from selected sources. Learn more about our {link}.",
     "link": "sources",
-    "license": "The internet is a big place! There are plenty of search portals and collections, hosting CC-licensed and public domain content, which we aren’t able to index or surface results from within the Openverse interface. These are important sources of CC-licensed and public domain content, and we want to make sure that you are able to find the best openly licensed materials possible, regardless of where they are located.",
+    "license": "The internet is a big place! There are plenty of search portals and collections, hosting CC-licensed and public domain content, which we aren’t able to index or surface results from within the {openverse} interface. These are important sources of CC-licensed and public domain content, and we want to make sure that you are able to find the best openly licensed materials possible, regardless of where they are located.",
     "content": "What content types can I search for using the Meta Search feature?",
-    "content-types": "Meta Search on Openverse currently supports image, audio, and video search on external sites.",
+    "content-types": "Meta Search on {openverse} currently supports image, audio, and video search on external sites.",
     "use": " Use ",
     "images": {
       "title": "How does the Meta Search feature work for indexed media types?",
       "demo-label": "Layout of Meta Search for indexed media type, like images",
-      "content": "Openverse supports built-in search for some media types, such as images. When you enter a query on the Openverse homepage, a page of results will be displayed. Scrolling to the bottom of the page, you have two choices:",
-      "continue": "{load-more}, and continue browsing in Openverse.",
+      "content": "{openverse} supports built-in search for some media types, such as images. When you enter a query on the {openverse} homepage, a page of results will be displayed. Scrolling to the bottom of the page, you have two choices:",
+      "continue": "{load-more}, and continue browsing in {openverse}.",
```

Comment:
Also missing the variable

https://github.com/WordPress/openverse-frontend/blob/15e9e2fd2015e45cdeb0e21e52e047a68e3da1a2/src/pages/meta-search.vue#L36